<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Lucida Console</family>
    <pointsize>36</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 36pt &quot;Lucida Console&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <widget class="QWidget" name="home" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_welcome">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>300</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Welcome to the Eyebot rPi.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_robios">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>30</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Robios Version: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_battery">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>70</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Battery: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_IP">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>100</y>
       <width>100</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>IP-Addr:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_hostname">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>115</y>
       <width>100</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Hostname:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_macaddr">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>130</y>
       <width>100</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Mac-Addr:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ssid">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>145</y>
       <width>100</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Wifi SSID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_wifiip">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>160</y>
       <width>100</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Console</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Wifi IP:</string>
     </property>
    </widget>
    <widget class="QWidget" name="menu_home" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>320</width>
       <height>45</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButton_info">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>80</width>
        <height>45</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>36</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Info</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_hardware">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>80</width>
        <height>45</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>36</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 0);</string>
      </property>
      <property name="text">
       <string>Hardware</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_software">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>80</width>
        <height>45</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>36</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 0);
</string>
      </property>
      <property name="text">
       <string>Software</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_demos">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>0</y>
        <width>80</width>
        <height>45</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>36</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>Demos</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="info" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>RPi Info</string>
     </property>
    </widget>
    <widget class="QWidget" name="menu_info" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>320</width>
       <height>45</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButton_home">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>80</width>
        <height>45</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>36</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_hardware_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>80</width>
        <height>45</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>36</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 0);</string>
      </property>
      <property name="text">
       <string>Hardware</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_software_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>80</width>
        <height>45</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>36</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 0);
</string>
      </property>
      <property name="text">
       <string>Software</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_demos_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>0</y>
        <width>80</width>
        <height>45</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>36</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>Demos</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tempBlack" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>300</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <zorder>tempBlack</zorder>
   <zorder>info</zorder>
   <zorder>home</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton_info</tabstop>
  <tabstop>pushButton_hardware</tabstop>
  <tabstop>pushButton_software</tabstop>
  <tabstop>pushButton_demos</tabstop>
  <tabstop>pushButton_home</tabstop>
  <tabstop>pushButton_hardware_2</tabstop>
  <tabstop>pushButton_software_2</tabstop>
  <tabstop>pushButton_demos_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
